packages/rich-text/src/plugins/Hyperlink/index.tsx

```diff
@@ -57,13 +57,29 @@ const styles = {
   }),
 };
 
-export function createHyperlinkPlugin(): SlatePlugin {
-  const keys: string[] = [INLINES.HYPERLINK, INLINES.ENTRY_HYPERLINK, INLINES.ASSET_HYPERLINK];
-
+export function createHyperlinkPlugin(sdk: FieldExtensionSDK): SlatePlugin {
   return {
-    renderElement: getRenderElement(keys),
-    pluginKeys: keys,
-    inlineTypes: getSlatePluginTypes(keys),
+    renderElement: getRenderElement(LINK_TYPES),
+    pluginKeys: LINK_TYPES,
+    inlineTypes: getSlatePluginTypes(LINK_TYPES),
+    onKeyDown: buildHyperlinkEventHandler(sdk),
+  };
+}
+
+export function buildHyperlinkEventHandler(sdk) {
+  return function withHyperlinkEvents(editor) {
+    return function handleKeyDown(event: KeyboardEvent) {
+      if (!editor.selection) return;
+      const isMod = event.ctrlKey || event.metaKey;
+      const K = 75;
+      if (isMod && event.keyCode === K) {
```

Comment:
I think it would be better if we transform `event.keyCode === K` into its own variable, something like `isK`.